src/convertToObject.js

```diff
@@ -1,18 +1,23 @@
 'use strict';
 
-/**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
- *
- * @param {string} sourceString
- *
- * @return {object}
- */
 function convertToObject(sourceString) {
-  // write your code here
+  const obj = {};
+
+  const modifiedArray = sourceString.split(';');
+
+  for (const line of modifiedArray) {
+    if (line.trim()) {
+      const devidedLine = line.split(':');
+
+      for (let i = 0; i < devidedLine.length; i++) {
+        devidedLine[i] = devidedLine[i].trim();
+      }
+
+      obj[devidedLine[0]] = devidedLine[1];
+    }
```

Comment:
```suggestion
      obj[devidedLine[0].trim()] = devidedLine[1].trim();
    }
```
to avoid redundant loop